app/models/users/setting.rb

```diff
@@ -13,6 +15,9 @@ class Setting < ApplicationRecord
     enum config_theme: { default: 0, minimal_light_theme: 1, night_theme: 2, pink_theme: 3,
                          ten_x_hacker_theme: 4 }
 
+    validates :brand_color1,
+              :brand_color2,
+              format: { with: HEX_COLOR_REGEXP, message: "is not a valid hex color" }
```

Comment:
Any chance that any of the existing values in the DB is invalid?

Reply:
Good question!

1. The migration used to be on `profiles` and was moved over here.
2. The current data in settings were migrated with a DUS from profiles.
3. As you said the data is entered via a color picker, so should always be of the form `#xxxxxx`.

It may still be worth double checking though.

Reply:
I checked, we have 3 users that would fail the validation. I'll talk to Andy Z.